galsim/photon_array.py

```diff
@@ -351,3 +351,109 @@ def applyTo(self, photon_array):
         tantheta = np.sqrt(np.square(sintheta) / (1. - np.square(sintheta)))
         dxdz[:] = tantheta * np.sin(phi)
         dydz[:] = tantheta * np.cos(phi)
+
+class PhotonDCR(object):
```

Comment:
I'm not sure those suggestions are clearer, although I did add "and optionally the chromatic dilation due to atmospheric seeing."  But I don't think most users will set the alpha parameter because it won't make sense for the galaxies, and you don't want to do it for your stars but not the galaxies if you plan to do any kind of WL analysis using PSFs drawn from the data.  (They wouldn't match up.)
So I'm inclined to keep the name be DCR-specific, rather than including the atmospheric seeing, even though it is technically an option.